components/AboutUs.js

```diff
@@ -0,0 +1,20 @@
+import Link from "next/link";
+import styled from "styled-components";
+
+export default function AboutUs() {
+    return (
+        <StyledLink href={"/aboutus"}>About us</StyledLink>
+    )
```

Comment:
Do you really need a separate Component for this? You could just just the styledLink directly this would reduce the complexity and increase the readability of your code.